pkg/input/joystick_controller.go

```diff
@@ -50,12 +80,19 @@ func (c *JoystickController) Open(deviceIndex int) bool {
 	c.buttonsPressed = make([]bool, c.numButtons)
 	c.buttonsReleased = make([]bool, c.numButtons)
 
+	log.Printf("Joystick #%d: opened. %s", c.joystick, c.Description())
 	c.findMapping()
 
 	return true
 }
 
 func (c *JoystickController) Close() {
+	log.Printf("Joystick #%d: closed", c.joystick)
```

Comment:
Only one joystickController at time can be connected to one device index. to be able to connect a new one to the same index, the first one need to be closed first